### PR TITLE
Refresh /partners/iot-device for canonical-v3

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Refresh /partners/iot-device for canonical-v3, see [design](https://app.zeplin.io/project/624ed20e0af92866116ea26d/screen/6254183bf20adf6e2dfda669) and [copydoc](https://docs.google.com/document/d/1qNEjihmySR6l6LZJ-SkSVTpaBSxuY8imoseN0OLkiMw/edit#)

## QA

- Compare against design and copydoc

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5156

### CODE REVIEW

**Review by mtruj013 — CHANGES_REQUESTED** [review]
Looks good @petesfrench, just a couple things:

File: templates/partners/iot-device.html [hunk 1]
```diff
@@ -18,11 +19,20 @@ <h1>IoT Device Programme</h1>
...
+      }}
     </div>
   </div>
 </section>
-<section class="p-strip is-deep">
+
+<section class="p-strip has-accent-border--top-left is-deep">
   <div class="u-fixed-width u-align--center">
     <p class="p-logo-section__title" style="max-width: none;">A selection of our IoT partners </p>
     <div class="p-logo-section">
```
[the remark this answers is not in the document]
Comment (petesfrench) [reply templates/partners/iot-device.html:38]: Will fix in other issue

File: templates/partners/iot-device.html [hunk 2]
```diff
@@ -18,13 +19,22 @@ <h1>IoT Device Programme</h1>
...
+        ) | safe
+      }}
     </div>
   </div>
 </section>
-<section class="p-strip is-deep">
+
+<section class="p-strip has-accent-border--top-left is-deep">
   <div class="u-fixed-width u-align--center">
-    <p class="p-logo-section__title" style="max-width: none;">A selection of our IoT partners </p>
```
Comment (mtruj013) [inline templates/partners/iot-device.html:27]: This is no longer centered:
Screenshot 2022-04-28 at 12 40 28

**Review by mtruj013 — APPROVED** [review]
LGTM, just one thing:

File: templates/partners/iot-device.html (hunk 2 above)
Comment (petesfrench) [reply templates/partners/iot-device.html:27]: Thank you and good catch!

**Pull request merged**